hwdb/60-sensor.hwdb

```diff
@@ -42,6 +42,12 @@
 # Sort by brand, model
 
 #########################################
+# Acer
+#########################################
+evdev:input:b0019v0000p0000e0000*
```

Comment:
That's not the correct syntax for this file, see the comments at the top of the file for the correct syntax.